baseframe/forms/auto.py

```diff
@@ -18,24 +18,28 @@ class ConfirmDeleteForm(Form):
     cancel = SubmitField(__(u"Cancel"))
 
 
-def render_form(form, title, message='', formid=None, submit=__(u"Submit"), cancel_url=None, ajax=False, with_chrome=True, autosave=False):
+def render_form(form, title, message='', formid=None, submit=__(u"Submit"), cancel_url=None, ajax=False, with_chrome=True, action='', autosave=False, last_revision_id=None):
```

Comment:
Action parameter default should be `action=None`. The name `last_revision_id` is verbose. This could be `draft_revision` instead.